examples/gh-pages/README.md

```diff
@@ -1,45 +0,0 @@
-# gh-pages Hello World example
```

Comment:
Do we want to leave this README and have a note linking to the new name similar to https://github.com/vercel/next.js/blob/canary/examples/with-material-ui/README.md

Reply:
I'm leaning towards no. Here's why: we have _so_ many examples now, we're almost reaching "folder debt" where I'd rather have a low probability 404 on `gh-pages` than harder scanning through the directly. I also instinctly searched for "github" and didn't find anything, so I almost missed this example entirely.

Open to other thoughts.